maro/__misc__.py

```diff
@@ -4,4 +4,4 @@
 
 __version__ = "0.1.1a11"
 
-__data_version__ = "0.1"
+__data_version__ = "0.2"
```

Comment:
Is the data version independent to the project version?

Reply:
yes